code/modules/antagonists/changeling/changeling_power.dm

```diff
@@ -34,6 +34,8 @@
 	var/active = FALSE
 	/// Does this ability stop working if you are burning?
 	var/disabled_by_fire = TRUE
+	/// Ability that may need to be bought first
+	var/list/prereq_ability
```

Comment:
This information should be shown in the Cellular Emporium UI - at least in a tooltip, but definitely disabling the button